### PR TITLE
completed React Stopwatch Component

### PR DESCRIPTION
**Pull request by CollinHovey**
Screenshot from 2022-07-14 20-08-33
Screenshot from 2022-07-14 20-08-37
Screenshot from 2022-07-14 20-08-38
Screenshot from 2022-07-14 20-08-39
Screenshot from 2022-07-14 20-08-41
Screenshot from 2022-07-14 20-08-45
ntent.com/102557430/179142349-0e3338b1-8464-4a8b-8cf8-a0c2f1115b74.png)

### CODE REVIEW

File: react-stopwatch-component/src/stopwatch.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      return <div className='container'>
+        <h1 className="watch">{this.state.counter}</h1>
+        <i onClick={this.handleClick} className="fa-solid fa-pause fa-4x"></i>
+      </div>;
+    } else {
+      return <div className='container'>
+        <h1 className="watch">{this.state.counter}</h1>
+        <i onClick={this.handleClick} className="fa-solid fa-play fa-4x"></i>
+      </div>;
+    }
```
Comment (scbowler) [inline react-stopwatch-component/src/stopwatch.jsx:33]: Note the only thing that changes between the two returns is the class on the `i` tag. So a better solution would be:
```JSX
let icon = 'fa-play';

if (this.state.on) {
  icon = 'fa-pause';
}

return (
  <div className='container'>
    <h1 className="watch">{this.state.counter}</h1>
    <i onClick={this.handleClick} className={`fa-solid ${icon} fa-4x`}></i>
  </div>
);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Good job getting this to work, but some refactoring needed. Let me know if you have any questions.

**Comment by CollinHovey** [comment]
ya this seems to be a hard thing for me, seeing the simpler way to write code

**Comment by scbowler** [comment]
> ya this seems to be a hard thing for me, seeing the simpler way to write code

That is common in new developers, but you'll get there! 😄

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**